src/System.Private.CoreLib/shared/System/Collections/Generic/ArraySortHelper.cs

```diff
@@ -0,0 +1,1117 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+/*============================================================
+**
+**
+** 
+**
+**
+** Purpose: class to sort arrays
+**
+** 
+===========================================================*/
```

Comment:
You could remove this header comment as it doesn't seem to be useful or perhaps just add it as summary on top of the class instead.